packages/ove-service-layout/docs/WORKING_EXAMPLE.md

```diff
@@ -0,0 +1,105 @@
+# Working example
+
+```json
+{
+  "hasVideos": false,
+  "ove-space": "http://localhost:3004/space-info",
```

Comment:
```suggestion
  "oveSpace": "http://localhost:3004/space-info",
```
Shall we always stick to camelCase within our JSON structures? That helps a lot especially when using these within JS.

Reply:
Please fix the same in other files - I am not making the same remark throughout.